lib/tapjoy/ldap/key/install.rb

```diff
@@ -0,0 +1,46 @@
+module Tapjoy
+  module LDAP
+    module Key
+      # Install key on localhost
+      class Install
+        def install
+          Tapjoy::LDAP::Key.get_keys_from_ldap.each do |user, values|
+            directory = directory(user)
+            FileUtils.mkdir_p(directory) unless File.exists?directory
+            authorized_keys_file = "#{directory}/authorized_keys"
+            keys = load_keys_from_file(authorized_keys_file)
+            insert_keys(authorized_keys_file, keys, values)
+          end
+        end
+
+        private
+        def opts
+          @opts ||= Trollop::options do
+              # Set help message
+              usage 'key install [options]'
```

Comment:
we can probably ditch [options] here so user don't think there are options they can access that aren't listed.